src/components/BarChart/BarChart.jsx

```diff
@@ -45,7 +46,7 @@ const BarChart = createClass({
 		MARGIN: {
 			top: 10,
 			right: 20,
-			bottom: 50,
+			bottom: 75,
```

Comment:
I don't think we want to change the default bottom margin since it should have already been tweaked to look good with horizontally oriented axis text.